codegenerator/cli/templates/dynamic/codegen/src/IO.res.hbs

```diff
@@ -17,8 +17,8 @@ let getEntityHistoryItems = entityUpdates => {
   ) => {
     let (optPreviousEventIdentifier, entityHistoryItems) = prev
 
-    let {eventIdentifier, shouldSaveHistory, entityUpdateAction, entityId} = entityUpdate
```

Comment:
I've refactored entity history in the in memory table to simplify things (and exposed a bug).

The inMemTable would hold "latestItem" and "history", where history did not include latest item. The problem is that each item would need to keep track of whether it should be saved to history or not, and the latest item would need to be added to history when requesting history.

Now it does not need to keep track of that, when calling "set" purely based on the "shouldSaveHistory" flag it either adds to history or not. The only minor knock is that you have a duplicate value at the head of "history" and in the "latestItem" position.